pkg/cli/cmd/credential/show/objectformats.go

```diff
@@ -17,34 +17,71 @@ limitations under the License.
 package show
 
 import (
+	"fmt"
 	"strings"
 
+	"github.com/radius-project/radius/pkg/cli/credential"
 	"github.com/radius-project/radius/pkg/cli/output"
+	"github.com/radius-project/radius/pkg/ucp/datamodel"
 )
 
 // credentialFormat function returns a FormatterOptions struct based on the credentialType parameter, which can
 // be either "azure" or "aws".
-func credentialFormat(credentialType string) output.FormatterOptions {
+func credentialFormat(credentialType string, providers credential.ProviderCredentialConfiguration) (output.FormatterOptions, error) {
```

Comment:
Is this function actually helping? It looks like we have to pass in the credential type, and now the credential type so we can switch between these four different options. It's essentially just accessing static data.  

Happy to for it to stay as-is or change it, do whatever seems most sensible.